software/wmla120_ansible/validate_firewall_setup.yml

```diff
@@ -1,15 +1,15 @@
 ---
 #If a firewall is enabled, the following default ports must be granted access on all management hosts for IBM Spectrum Conductor Deep Learning Impact: 9243, 9280, 5000, 5001, 27017, and 6379. If you change these ports after installation, make sure to update firewall rules accordingly.
 #
+#
+- name: Include vars of envs_spectrum_conductor_dli_template.yml into the 'ports' variable
+  include_vars:
```

Comment:
This needs to be the working copy of the file (envs_spectrum_conductor_dli.yml) not the template. 
I didn't think of this sooner, but the user isn't prompted to edit the environment variable files until install. (See the install code at or around line 2037 in wmla120.py listed below for reference).  These function calls will need to be moved into init-clients before the validation checks

        _set_spectrum_conductor_install_env(self.sw_vars['ansible_inventory'],
                                            'spark')
        _set_spectrum_conductor_install_env(self.sw_vars['ansible_inventory'],
                                            'dli', ana_ver)

Reply:
- Addressed .yml file in PR#514, now gets working copy of envs_spectrum_conductor_dli

- Addressed function call to be moved to init-clients stage in PR#515